vuln-reach/src/javascript/lang/symbol_table.rs

```diff
@@ -109,11 +109,20 @@ impl<'a> SymbolTableBuilder<'a> {
                 // Statement block and program nodes are the only node types
                 // that create a new scope.
                 self.push_scope(node);
-                if let Some(params) = node.prev_named_sibling() {
-                    if params.kind() == "formal_parameters" {
-                        self.visit(params);
+
+                let mut cur_node = node;
+                loop {
+                    cur_node = match cur_node.prev_named_sibling() {
```

Comment:
I feel like it would be much cleaner to make this into a `while let Some` loop and just assign `cur_node` at the end of its body.